great_expectations/expectations/core/expect_column_values_to_be_in_set.py

```diff
@@ -185,92 +188,59 @@ class ExpectColumnValuesToBeInSet(ColumnMapExpectation):
     }
 
     @classmethod
-    def _atomic_prescriptive_template(
+    def _prescriptive_template(
         cls,
-        configuration: Optional[ExpectationConfiguration] = None,
-        result: Optional[ExpectationValidationResult] = None,
-        runtime_configuration: Optional[dict] = None,
-    ) -> Tuple[str, dict, Union[dict, None]]:
-        renderer_configuration = RendererConfiguration(
-            configuration=configuration,
-            result=result,
-            runtime_configuration=runtime_configuration,
-        )
-        renderer_configuration.add_param(
-            name="column",
-            schema_type="string",
-        )
-        renderer_configuration.add_param(
-            name="value_set",
-            schema_type="array",
-        )
-        renderer_configuration.add_param(
-            name="mostly",
-            schema_type="number",
-        )
-        renderer_configuration.add_param(
-            name="mostly_pct",
-            schema_type="string",
-        )
-        renderer_configuration.add_param(
-            name="parse_strings_as_datetimes",
-            schema_type="boolean",
-        )
-        renderer_configuration.add_param(
-            name="row_condition",
-            schema_type="string",
-        )
-        renderer_configuration.add_param(
-            name="condition_parser",
-            schema_type="string",
+        renderer_configuration: RendererConfiguration,
+    ) -> RendererConfiguration:
+        add_param_args = (
+            ("column", ParamSchemaType.STRING),
+            ("value_set", ParamSchemaType.ARRAY),
+            ("mostly", ParamSchemaType.NUMBER),
+            ("parse_strings_as_datetimes", ParamSchemaType.BOOLEAN),
+            ("row_condition", ParamSchemaType.STRING),
+            ("condition_parser", ParamSchemaType.STRING),
         )
+        for name, schema_type in add_param_args:
+            renderer_configuration.add_param(name=name, schema_type=schema_type)
 
-        params: RendererParams
-        params = renderer_configuration.params
-
-        if not params.value_set or len(params.value_set.value) == 0:
-            values_string = "[ ]"
-        else:
-            for i, v in enumerate(params.value_set.value):
-                renderer_configuration.add_param(
-                    name=f"v__{str(i)}", schema_type="string", value=v
-                )
+        params: RendererParams = renderer_configuration.params
+        template_str = ""
```

Comment:
Doubt this is a real bottleneck but should we be using a string builder? Perhaps a list and then a `" ".join(...)`? Probably not worth investigating unless we know it's a performance issue.

Reply:
It would be better, but for now it will need to wait considering all 300+ Expectation renderers do this.